src/theme/components/button.ts

```diff
@@ -0,0 +1,53 @@
+import { SystemStyleObject } from '@chakra-ui/theme-tools';
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+import { sizes, typography } from '../shared';
+
+const baseStyle: SystemStyleObject = {
+  borderRadius: 'sm',
+  px: 'lg',
+};
+
+const buttonSizes = {
+  md: {
+    height: sizes.md,
+    ...typography.heading5,
+  },
+  lg: {
+    height: sizes.lg,
+    ...typography.heading4,
+  },
+  xl: {
+    height: sizes.xl,
+    ...typography.heading4,
+  },
+};
+
+const variants = {
+  primary: {
+    bg: 'brand.primary',
+    color: 'gray.900',
+    boxShadow: 'button',
+    _hover: {
+      filter: 'brightness(1.1)',
+    },
+    _active: {
+      bg: 'brand.100',
```

Comment:
Is this one the same in as/ms or do we need a separate token?

Reply:
At the moment, I don't have any info for active bg token of MS24, so I leave as it is was configured in AS24. We can come back later if Figma get updated